server/api/utils/singletons/k8s.py

```diff
@@ -540,6 +540,64 @@ def delete_secrets(
         self.v1api.delete_namespaced_secret(secret_name, namespace)
         return mlrun.common.schemas.SecretEventActions.deleted
 
+    def create_configmap(
```

Comment:
use `raise_for_status_code` or `mlrun.errors.err_for_status_code` to map the exception to an mlrun exception